src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -315,12 +329,136 @@ private static void printLocales() {
             // print columns of 8
             if ((i + 1) % 8 == 0) {
                 ostream.println();
-                ostream.print(INDENT + INDENT);
+                ostream.print(TWOINDENT);
+            }
+        }
+        ostream.println();
+    }
+
+    private static void printSecuritySettings(String arg) {
+        switch (arg) {
+            case "properties" -> printSecurityProperties();
+            case "providers"  -> printSecurityProviderConfig(true);
+            case "tls"        -> printSecurityTLSConfig(true);
+            case "all"        -> printAllSecurityConfig();
+            default           -> {
+                ostream.println("Unrecognized security sub-option. Valid values are " +
+                        "\"all\", \"properties\", \"providers\", \"tls\". See \"java -X\"");
+                ostream.println("Printing all security settings\n");
+                printAllSecurityConfig();
+            }
+
+        }
+    }
+
+    // A non-verbose description of some core security configuration settings
+    private static void printSecuritySummarySettings() {
+        ostream.println("Security settings summary: " + "\n" +
+                INDENT + "Use \"-XshowSettings:security\" for verbose details\n");
```

Comment:
I think it might be better to delegate to `java -X` help output actually. tweaked code in this area. (I could see the subcommand options becoming inaccurate if we add more such subcommands in the future). -X help output should be our reference